src/messages/lang_fr-FR.json

```diff
@@ -7,16 +7,16 @@
     "lang-nl-NL": "Nederlands",
     "lang-sv-SE": "Svenska",
 
-    "navButtonTooltip": "Sélectionnez pour afficher d'autres langues",
+    "navButtonTooltip": "Sélectionner pour afficher d'autres langues",
     "lang-en-US-tooltip": "Select for English",
-    "lang-fr-FR-tooltip": "Sélectionnez pour le français",
+    "lang-fr-FR-tooltip": "Sélectionner pour le français",
```

Comment:
The french change for language buttons and tooltips should also be made into other "lang_XX.json" files such as lang_en-US.json and lang_es-MX.json.